nodejs-react-onclickoutside/nodejs-react-onclickoutside.spec

```diff
@@ -0,0 +1,46 @@
+%global npm_name react-onclickoutside
+%global enable_tests 0
+
+%{?nodejs_find_provides_and_requires}
+
+Name: nodejs-%{npm_name}
+Version: 6.6.3
+Release: 1%{?dist}
+Summary: An onClickOutside wrapper for React components
+License: MIT
+URL: https://github.com/Pomax/react-onclickoutside
+Source0: https://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
+BuildRequires: nodejs-packaging
+BuildArch:  noarch
+ExclusiveArch: %{nodejs_arches} noarch
+
+%{?nodejs_find_provides_and_requires}
+
+%define npm_cache_dir /tmp/npm_cache_%{name}-%{version}-%{release}
```

Comment:
This appears to be unused for non-bundled NPMs, so for cleanness sake's we should remove references to it (and update npm2rpm)